web-api/src/business/useCases/document/batchDownloadDocketEntriesInteractor.ts

```diff
@@ -19,14 +20,16 @@ export type DownloadDocketEntryRequestType = {
 export const batchDownloadDocketEntriesInteractor = async (
   applicationContext: ServerApplicationContext,
   downloadDocketEntryRequestInfo: DownloadDocketEntryRequestType,
+  authorizedUser: UnknownAuthUser,
 ) => {
   try {
     await batchDownloadDocketEntriesHelper(
       applicationContext,
       downloadDocketEntryRequestInfo,
+      authorizedUser,
     );
   } catch (error: any) {
-    const { userId } = applicationContext.getCurrentUser();
+    const { userId } = authorizedUser;
```

Comment:
Should there be a check that the user is authorized (or at least logged in?)

Reply:
Good question. For this one we kept the same functionality as existed before but would be happy to add an if check and a test.

Reply:
Thanks, Zach.  Let's punt in the interest of minimizing functionality changes for now...